packages/https-proxy/lib/ca.js

```diff
@@ -269,7 +269,7 @@ class CA {
   }
 
   writeCAVersion () {
-    return fs.outputFileAsync(this.getCAVersionPath(), CA_VERSION)
+    return fs.outputFileAsync(this.getCAVersionPath(), String(CA_VERSION))
```

Comment:
Node is stricter now, we can't directly write a number to a file, has to be a string or buffer